docs/sql/system.txt

```diff
@@ -688,6 +688,34 @@ _node
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+recovery
+--------
+
++------------------------------------------+----------------------------------------------------+-------------+
+| Column Name                              | Description                                        | Return Type |
++==========================================+====================================================+=============+
+| ``recovery``                             | Represents recovery statistic of the particular    | ``Object``  |
+|                                          | index.                                             |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']``                     | Statistics about physical index recovery in bytes. | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['total_bytes']``      | Total number of bytes in the index.                | ``Long``    |
```

Comment:
it should reflect https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-recovery.html